tests/Unit/RemoteOutputSizeLimitsTest.php

```
@@ -0,0 +1,124 @@
+<?php
+
+use App\Actions\Proxy\GetProxyConfiguration;
+use App\Jobs\ScheduledTaskJob;
+use App\Livewire\Project\Shared\GetLogs;
+use App\Livewire\Server\Proxy\DynamicConfigurations;
+use App\Models\Application;
+use App\Models\Server;
+
+function remoteOutputSource(string $path): string
+{
+    return file_get_contents(__DIR__.'/../../'.$path);
```

PHPStan (2.2.7)

[warning] Function remoteOutputSource() should return string but returns string|false.


(return.type)